### PR TITLE
1 set up project

### LINKED ISSUE #1: set-up-project
### Issue: set-up-project
- I will create the following files: index.html, style.css, and index.js 
- I will add a fontawesome icon.
- I will add a checkbox for the accomplished tasks
- I will add linters like Webhint, StyleLint, EsLint
- I will add a Form and create inside inputs so the user can add his/her tasks there

### Goal:
The goal of this project is to understand how to set up the webpack and its dependencies, 
also to understand and use ES6, construct an array of objects, loops, and the use of DOM.

### PR DESCRIPTION
**Pull request by IvonneBenitesRodriguez**
### About this project
- It is a todo-list project basically to put in practice JS functionalities.
- It has a remove and add section
- It has a HTML , JS and CSS files also.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+taskInput.addEventListener('keydown', (event) => {
+  if (event.key === 'Enter') {
+    event.preventDefault();
+    const userInput = taskInput.value;
+    taskInput.value = '';
+    displayUserInput(userInput);
+  }
+});
```
Comment (MarkYegon7) [inline app.js:63]: Nice work :100: 

- You followed the correct javascript principles.
- Use of proper ES6 syntax.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                    <i class="fa-solid fa-arrow-left"></i>
+                </form>
+                
+                <span id="outputList"></span>
+
+                <div class="btn-clear">    
+                     <input type="button" id="btn-clear-completed" value="Clear all completed">
+                </div>
+                
+        </section>
```
Comment (MarkYegon7) [inline index.html:31]: Good indentation.

**Review by MarkYegon7 — COMMENTED** [review]
## Status Approved :recycle: 
Good work

File: app.js (hunk 1 above)
Comment (IvonneBenitesRodriguez) [reply app.js:63]: Hello @MarkYegon7 ,
Thank you so much for the kindness, 
I appreciate it, 
Best regards,

**Pull request merged**